### PR TITLE
7005 rasa interactive cannot write non ascii character

### LINKED ISSUE #7005: Rasa interactive cannot write non-ascii character due to the encoing
**Rasa version**: 1.10, 2.*

**Rasa SDK version** (if used & relevant):

**Rasa X version** (if used & relevant):

**Python version**: 3.7.6

**Operating system** (windows, osx, ...): Windows

**Issue**:
The rasa interactive write file write with the default encoding, for windows python, it may cause error if the user type utf-8 character.

**Error (including full traceback)**:
```
Traceback (most recent call last):
  File "e:\env\anaconda\envs\rasa\lib\site-packages\rasa\core\training\interactive.py", line 1465, in record_messages
    plot_file,
  File "e:\env\anaconda\envs\rasa\lib\site-packages\rasa\core\training\interactive.py", line 949, in _predict_till_next_listen
    unconfirmed=[ActionExecuted(action_name)],
  File "e:\env\anaconda\envs\rasa\lib\site-packages\rasa\core\training\interactive.py", line 1394, in _plot_trackers
    write_dot(graph, output_file)
  File "<decorator-gen-781>", line 2, in write_dot
  File "e:\env\anaconda\envs\rasa\lib\site-packages\networkx\utils\decorators.py", line 240, in _open_file
    result = func_to_be_decorated(*new_args, **kwargs)
  File "e:\env\anaconda\envs\rasa\lib\site-packages\networkx\drawing\nx_pydot.py", line 47, in write_dot
    path.write(P.to_string())
UnicodeEncodeError: 'cp950' codec can't encode character '\u5605' in position 505: illegal multibyte sequence
```

**Command or request that led to error**:
```

```

**Content of configuration file (config.yml)** (if relevant):
```yml

```

**Content of domain file (domain.yml)** (if relevant):
```yml

```

**Solution**:
change rasa/core/training/interactive.py : 1394
```
   write_dot(graph, output_file)
```
to
```
    output_file = open(output_file, 'w', encoding='utf-8')
    write_dot(graph, output_file)
    output_file.close()
```

### PR DESCRIPTION
**Pull request by joaoCeilandia**
**Proposed changes**:
- fix #7005 

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @desmarchris will take a look at it as soon as possible ✨

**Comment by HotThoughts** [comment]
Hey, the failing step wait for doc tests in CI workflow has fixed. The PR should be ready to review when the fix merged into your PR. 😊

**Comment by HotThoughts** [comment]
Seems like the fix does not work with the situation when updating from the base branch I will fix this soon!

**Comment by HotThoughts** [comment]
👋 Hey, the error is fixed. Please update the branch.

**Comment by joaoCeilandia** [comment]
Thanks! @HotThoughts

**Comment by HotThoughts** [comment]
@RasaHQ/enable-squad maybe one of you can review this PR?

**Comment by twerkmeister** [comment]
@HotThoughts will look into it!

File: changelog/7005.bugfix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Add encoding ´utf-8´ to write_dot funcion
```
Comment (twerkmeister) [inline changelog/7005.bugfix.md:1]: As this goes into a big change log file, the following gives a bit more context:
```suggestion
Fixed a bug for plotting trackers with non-ascii texts during interactive training by enforcing utf-8 encoding
```

File: rasa/core/training/interactive.py [hunk 2]
```diff
@@ -1412,7 +1412,9 @@ async def _plot_trackers(
 
     from networkx.drawing.nx_pydot import write_dot
 
+    output_file = open(output_file, "w", encoding="utf-8")
     write_dot(graph, output_file)
+    output_file.close()
```
Comment (twerkmeister) [inline rasa/core/training/interactive.py:1417]: Doesn't override existing variable, briefer, no closing necessary:
```suggestion
    with open(output_file, "w", encoding="utf-8") as f:
        write_dot(graph, f)
```

**Review by twerkmeister — CHANGES_REQUESTED** [review]
Very cool, good find and thanks for taking the initative on this one 👍 ! Just added small suggestions that you can use directly.

**Comment by joaoCeilandia** [comment]
Thanks for the feedback @twerkmeister !

**Review by twerkmeister — APPROVED** [review]
👍

**Pull request merged**